File: 9-objective-c/MyCompany/MyCompany/ViewController.m
//
//  ViewController.m
//  MyCompany
//
//  Created by Mehmet Tarhan on 20/02/2022.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}


- (IBAction)showName:(id)sender {
    [self.nameLabel setText:@"My name is Objective-C"];
}

@end

File: 4-core-location-mapkit/Walk2Draw/Walk2Draw/ViewController.swift
//
//  ViewController.swift
//  Walk2Draw
//
//  Created by Mehmet Tarhan on 14/04/2022.
//

import UIKit

class ViewController: UIViewController {
    var locationProvider: LocationProvider?

    override func viewDidLoad() {
        super.viewDidLoad()

        view.backgroundColor = .systemGreen

        locationProvider = LocationProvider()
    }
    
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        
        locationProvider?.start()
    }
}
